packages/core/src/compiler/Substance.ts

```
@@ -241,122 +261,723 @@ const stringType: TypeConsApp<A> = {
+): CheckerResult<(Decl<A> | Bind<A>)[]> => {
+  const declBind = stmt;
+  const { type, variable, expr } = declBind;
+
+  const decl: Decl<A> = {
+    ...declBind,
+    tag: "Decl",
+    name: variable,
+  };
+
+  if ("expr" in decl) {
+    delete decl.expr;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)